Doc-restructure (#151)
bank-vaults/bank-vaults.dev: content/docs/_index.md

> Bank-Vaults provides the following tools for Hashicorp Vault to make its use easier and more automated:

Reviewer: ```suggestion
We provide the following tools for Hashicorp Vault to make its usage easier and more automated:
```

Reviewer: The main idea is understanding what `bank-vaults` represents across the docs: 
- as a project, it offers multiple tools
- as a repo, only the CLI tool

> - [Bank-Vaults CLI tool]({{< relref "/docs/cli-tool/_index.md" >}}) makes working with Hashicorp Vault easier. For example, it can automatically initialize, unseal, and configure Vault.

Reviewer: ```suggestion
- [Bank-Vaults CLI]({{< relref "/docs/cli-tool/_index.md" >}}) makes working with Hashicorp Vault easier. For example, it can automatically initialize, unseal, and configure Vault.
```

> - [Vault operator]({{< relref "/docs/operator/_index.md" >}}) is a Kubernetes operator that helps you operate Hashicorp Vault in a Kubernetes environment.

Reviewer: ```suggestion
- [Vault Operator]({{< relref "/docs/operator/_index.md" >}}) is a Kubernetes operator that helps you operate Hashicorp Vault in a Kubernetes environment.
```

> - [Vault secrets webhook]({{< relref "/docs/mutating-webhook/_index.md" >}}) is a mutating webhook for injecting secret directly into Kubernetes pods.

Reviewer: ```suggestion
- [Vault Secrets Webhook]({{< relref "/docs/mutating-webhook/_index.md" >}}) is a mutating webhook for injecting secrets directly into Kubernetes pods, config maps and custom resources.
```

- [Vault SDK]({{< relref "/docs/go-library/_index.md" >}}) is a Go client wrapper for the official Vault client with automatic token renewal, built-in Kubernetes support, and a dynamic database credential provider. It makes it easier to work with Vault when developing your own Go applications.

![Bank-Vaults overview](/docs/images/bank-vault-overview.png)

> Bank-Vaults also provides Helm charts for installing the various components, and a collection of scripts to support advanced features (for example, dynamic SSH).

Reviewer: ```suggestion
In addition, we also provide Helm charts for installing various components, as well as a collection of scripts to support advanced features (for example, dynamic SSH).
```

> {{% include-remote-mdsnippet "https://github.com/bank-vaults/vault-operator/raw/main/VERSIONS.md" %}}

Reviewer: This table is not properly formatted. Can we have a PR to fix this?
